CppCustomVisualizer2/dll/ChildVisualizer.cpp

```diff
@@ -20,7 +20,6 @@ HRESULT CChildVisualizer::Initialize(
 HRESULT CChildVisualizer::CreateEvaluationResult(
     _In_ DkmString* pName,
     _In_ DkmString* pFullName,
-    _In_ DkmString* pType,
```

Comment:
>_In_ [](http://example.com/codeflow?start=4&length=4)

Since this is a sample, I might suggest doing `_In_opt_` instead. Really, this sample should be providing the type.

Reply:
Oh got it!! Thanks for the tip, didn't think about that